lib/Importer/caffe2.proto

```diff
@@ -126,6 +126,10 @@ message QTensorProto {
   repeated int32 data = 6 [packed = true];
   optional string name = 7;
   optional TensorProto.DataType data_type = 8 [default = INT32];
+  repeated double scales = 9;
+  repeated double biases = 10;
+  optional int32 axis = 11;
+  optional bool is_multiparam = 12 [default = false];
```

Comment:
```
is_multiparam
```
I assume it refers to multiple scales/offsets? Can it have more explicit name)?

Reply:
Yeah, any suggestions? since this is QTensorProto already, so multiparam cannot indicate anything else...

Reply:
Guess I will just use this name... This is in QTensorProto already, so multiparam cannot mean anything else other than the quantization param...

Reply:
I think I can add some comments here in case someone forgot what it is